thrust/thrust/system/cuda/detail/core/util.h

```diff
@@ -176,7 +157,21 @@ struct specialize_plan_impl_match<P, typelist<SM, SMs...>>
     : ::cuda::std::conditional<has_sm_tuning<P, SM>::value, P<SM>, specialize_plan_impl_match<P, typelist<SMs...>>>::type
 {};
 
-template <template <class> class Plan, class SM = THRUST_TUNING_ARCH>
+#ifdef _NVHPC_CUDA
+#  if (__NVCOMPILER_CUDA_ARCH__ >= 600)
+#    define _THRUST_TUNING_ARCH sm60
```

Comment:
those are old

Reply:
I know, but they are still in use until Thrust kernels are migrated to CUB.